src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/ResettableCompletionSource.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net.Quic.Implementations.MsQuic.Internal
     /// A resettable completion source which can be completed multiple times.
     /// Used to make methods async between completed events and their associated async method.
     /// </summary>
-    internal sealed class ResettableCompletionSource<T> : IValueTaskSource<T>, IValueTaskSource
+    internal class ResettableCompletionSource<T> : IValueTaskSource<T>, IValueTaskSource
```

Comment:
You unsealed this, but I don't actually see where you are deriving from it? Did I miss something?

Reply:
This is a mistake, thanks for noticing.